trustkit/src/main/java/com/datatheorem/android/trustkit/TrustKit.java

```diff
@@ -166,38 +166,41 @@
  *         </network-security-config>
  *     }
  * </pre>
- *
  */
 public class TrustKit {
 
     protected static TrustKit trustKitInstance;
+    public static boolean failOnInvalidDomain = true;
 
     private final TrustKitConfiguration trustKitConfiguration;
 
     protected TrustKit(@NonNull Context context,
-                       @NonNull TrustKitConfiguration trustKitConfiguration) {
+        @NonNull TrustKitConfiguration trustKitConfiguration) {
         this.trustKitConfiguration = trustKitConfiguration;
 
         // Setup the debug-overrides setting if the App is debuggable
         // Do not use BuildConfig.DEBUG as it does not work for libraries
         boolean isAppDebuggable = (0 !=
-                (context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE));
-        Set<Certificate> debugCaCerts = null;
+            (context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE));
+        Set<Certificate> trustAnchors = null;
         boolean shouldOverridePins = false;
         if (isAppDebuggable) {
-            debugCaCerts = trustKitConfiguration.getDebugCaCertificates();
-            if (debugCaCerts != null) {
+            trustAnchors = trustKitConfiguration.getDebugCaCertificates();
+            if (trustAnchors != null) {
                 TrustKitLog.i("App is debuggable - processing <debug-overrides> configuration.");
             }
             shouldOverridePins = trustKitConfiguration.shouldOverridePins();
         }
 
+        if (trustAnchors == null) {
+            trustAnchors = trustKitConfiguration.getTrustAnchorsCertificates();
+        }
```

Comment:
This trust anchors would overwrite debug trust anchors, it basically makes debug ca certs unused. Is that what we want or do we maybe want to merge those two or maybe use trust anchors only in non-debug builds?